OpenEdXMobile/src/main/java/org/edx/mobile/util/JavaUtil.java

```diff
@@ -82,7 +82,7 @@ public static Double formatDoubleValue(@NonNull Double value, int places) {
      */
     @NonNull
     public static String truncateString(@NonNull String string, int maxChars) {
-        if (string.length() > maxChars) {
+        if (string != null && string.length() > maxChars) {
```

Comment:
Sorry! This is my bad. My app was crashing at this line because we did not update the blocks API on our platform as introduced in [edx-platform#14233](https://github.com/edx/edx-platform/pull/14233/files). We can revert this change and edit the platform required to run the app to `Ficus` in [README.rst](https://github.com/edx/edx-app-android/blob/master/README.rst)